ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -39,15 +39,15 @@ import {
   ButtonSecondary,
   ButtonSecondarySize,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
   BannerBase,
   Icon,
   IconName,
   IconSize,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import { TextFieldSearch } from '../../component-library/text-field-search/deprecated';
 import { ADD_POPULAR_CUSTOM_NETWORK } from '../../../helpers/constants/routes';
 import { getEnvironmentType } from '../../../../app/scripts/lib/util';
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 4 41 45 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e9520e53-7d12-4340-b125-de8150cb98e2">

### After

<img width="1512" alt="Screenshot 2023-12-10 at 4 41 55 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/9572d220-8cf5-4fa0-86ae-5fab62a10a7a">